CHANGELOG.md

```diff
@@ -47,6 +47,10 @@ SPDX-License-Identifier: LGPL-3.0-or-later
         - The assembler bindings now also accept affordances and `DBCOption`, and they are also renamed to simply `matrix`, `vector`
           and `scalar`.
         - The assemblers also export the binding functions to bind the assemblers.
+- Added the notion of manipulating the assemblers.
```

Comment:
This is a bit vague can you just state that there is a new assembler that wraps an existing assembler and lets manipulate single degrees?

Reply:
Also mention the new class name.